docs/docs/HowTo/subscriptions/My-subscription-requests.md

```diff
@@ -0,0 +1,52 @@
+# My Subscription Requests
+
+To access the **My Acl Requests** page, navgiate to the **Requests -\>
+My Acl Requests**. On this page, you can see all the requests created by
+you or your team, along with an audit log and the history of your
+subscription requests.
+
+The requests are displayed in chronological order by *Date Requested*,
+with the most recent request at the top and the oldest request at the
+bottom. Additionally, you can check the current status of any request
+you and your team have made on this page.
+
+## Filtering the requests
+
+For easy navigation use the dropdown to filter the requests which you
+are viewing by the status of the request.
+
+-   
+
+    all
+
+    :   View all requests that your team has created.
+
```

Comment:
leading : in each line
![image](https://github.com/aiven/klaw-docs/assets/121855584/78ef4f00-00bd-45c0-97d9-10f528d94f21)